react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -1,5 +1,8 @@
 package org.wordpress.mobile.ReactNativeGutenbergBridge;
 
+import android.os.Bundle;
+import android.os.Parcelable;
```

Comment:
Not sure if we need import of `Parcelable` here

Reply:
You're right. I missed this one too once I changed a signature below